src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -0,0 +1,230 @@
+          style={{ width: '270px', height: '162px' }}
+          title={''}
+          description={i18n.translate('workspace.initial.card.muchMore.description', {
+            defaultMessage: 'And much more...',
+          })}
+        />
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  const content = (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <EuiTitle size="l">
+          <h1>
+            {i18n.translate('workspace.initial.title', {
+              defaultMessage: 'Getting started with OpenSearch',
+            })}
+          </h1>
+        </EuiTitle>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} style={{ maxWidth: '650px' }}>
+        <EuiText grow={false}>
+          {i18n.translate('workspace.initial.description', {
+            defaultMessage:
+              'OpenSearch is a flexible, scalable, open-source way to build solutions for data-intensive search and analytics applications. Explore, enrich, and visualize your data, using developer-friendly tools and powerful integrations for machine learning, data processing, and more.',
+          })}
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} className="eui-displayInline">
+        <EuiButtonEmpty
+          href="https://www.opensearch.org/"
+          iconType="popout"
+          iconSide="right"
+          flush="left"
+          data-test-subj="workspace-initial-button-openSearch"
+        >
+          <EuiText>
+            {i18n.translate('workspace.initial.button.openSearch', {
+              defaultMessage: 'Learn more from documentation and more.',
+            })}
+          </EuiText>
+        </EuiButtonEmpty>
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false} style={{ maxWidth: '540px' }}>
+        <EuiPanel color="subdued" borderRadius="none" hasShadow={false} hasBorder={false}>
+          <EuiText>
+            <EuiIcon type="dashboardApp" size="l" />
+            &nbsp;&nbsp;Explore live demo environment at{' '}
+            <EuiLink href="https://future.playground.opensearch.org/">
+              playground.opensearch.org
+            </EuiLink>
+          </EuiText>
+        </EuiPanel>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>{cards}</EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  return (
+    <EuiPage style={{ minHeight: '100vh', display: 'flex' }}>
+      <EuiPageBody>
+        <EuiFlexGroup direction="column" justifyContent="center" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiImage size="l" alt="OpenSearch" src={logos.OpenSearch.url} />
+          </EuiFlexItem>
+          <EuiSpacer />
+          <EuiFlexItem grow={false} style={{ width: '1122px' }} className="eui-displayInline">
+            <EuiPageContent
+              style={{
+                backgroundImage: `url(${img})`,
+                backgroundSize: '490px 380px',
+                backgroundRepeat: 'no-repeat',
+                backgroundPosition: 'right 10px top 30px',
```

Comment:
The position looks weird to me, Is it necessary?

Reply:
Yes, the background should be post in a specific location.